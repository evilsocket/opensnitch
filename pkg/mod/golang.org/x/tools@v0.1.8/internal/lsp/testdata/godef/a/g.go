package a

import "time"

// dur is a constant of type time.Duration.
const dur = 15*time.Minute + 10*time.Second + 350*time.Millisecond //@dur,hoverdef("dur", dur)
