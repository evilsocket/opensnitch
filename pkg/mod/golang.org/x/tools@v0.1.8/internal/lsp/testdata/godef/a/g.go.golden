-- dur-hoverdef --
```go
const dur time.Duration = 910350000000 // 15m10.35s
```

dur is a constant of type time\.Duration\.
